app/client/templates/views/dashboard.html

```diff
@@ -52,9 +52,6 @@
             {{#each wallets}}
                 {{> elements_account account=_id wallets=true}}
             {{/each}}
-            {{#each (wallets true)}}
```

Comment:
The idea of this was to show the disabled (or watch only) wallets in a second row.
It seems that wallets don't have the `disabled` property anymore, which they used to have?! Who removed it and why?